### PR TITLE
update uuid to v10

### PR DESCRIPTION
**Pull request by bobslavtriev**

### CODE REVIEW

**Comment by PurpShell** [comment]
why? due to pseudo-randomness? or due to dependency chain? what about speed? Is uuid slower? Is the crypto module slower?

**Comment by PurpShell** [comment]
> > why? due to pseudo-randomness? or due to dependency chain? what about speed? Is uuid slower? Is the crypto module slower?
> 
> The old version of UUID has been discontinued, instead of updating it, just use crypto, so you don't need to install additional libs

will use v5

File: package.json [hunk 1]
```diff
@@ -56,7 +56,6 @@
     "node-cache": "^5.1.2",
     "pino": "^7.0.0",
     "protobufjs": "^7.2.4",
-    "uuid": "^9.0.0",
     "ws": "^8.13.0"
```
Comment (purpshell) [inline package.json:60]:
```suggestion
    "uuid": "^10.0.0",
    "ws": "^8.13.0"
```

File: src/Utils/auth-utils.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
-import { randomBytes } from 'crypto'
+import { randomBytes, randomUUID } from 'crypto'
 import NodeCache from 'node-cache'
 import type { Logger } from 'pino'
-import { v4 as uuidv4 } from 'uuid'
 import { DEFAULT_CACHE_TTLS } from '../Defaults'
```
Comment (purpshell) [inline src/Utils/auth-utils.ts]:
```suggestion
import { v4 as uuidv4 } from 'uuid'
import { DEFAULT_CACHE_TTLS } from '../Defaults'
```

File: src/Utils/auth-utils.ts [hunk 3]
```diff
@@ -1,7 +1,6 @@
-import { randomBytes } from 'crypto'
+import { randomBytes, randomUUID } from 'crypto'
```
Comment (purpshell) [inline src/Utils/auth-utils.ts]:
```suggestion
import { randomBytes } from 'crypto'
```

File: src/Utils/auth-utils.ts [hunk 4]
```diff
@@ -209,8 +208,8 @@ export const initAuthCreds = (): AuthenticationCreds => {
 			unarchiveChats: false
 		},
 		// mobile creds
-		deviceId: Buffer.from(uuidv4().replace(/-/g, ''), 'hex').toString('base64url'),
-		phoneId: uuidv4(),
+		deviceId: Buffer.from(randomUUID().replace(/-/g, ''), 'hex').toString('base64url'),
+		phoneId: randomUUID(),
```
Comment (purpshell) [inline src/Utils/auth-utils.ts]:
```suggestion
		deviceId: Buffer.from(uuidv4().replace(/-/g, ''), 'hex').toString('base64url'),
		phoneId: uuidv4(),
```

**Review by PurpShell — CHANGES_REQUESTED** [review]
just upgrade to v10

**Pull request merged**